src/report/entities/feedback.repository.ts

```diff
@@ -0,0 +1,32 @@
+import { User } from '../../users/entities/user.entity';
+import { EntityRepository, Repository } from 'typeorm';
+import { FeedbackCreationDto } from '../dto/feedback.dto';
+import { Feedback } from './feedback.entity';
+
+@EntityRepository(Feedback)
+export class FeedbackRepository extends Repository<Feedback> {
+  async createFeedback(
+    feedbackDto: FeedbackCreationDto,
+    user: User,
+  ): Promise<void> {
+    const { feedback_body, feedback_choice } = feedbackDto;
+    const feedback = this.create();
+    feedback.feedback_choice = feedback_choice;
+    feedback.feedback_body = feedback_body;
+    feedback.user = user;
+    await this.save(feedback);
```

Comment:
Please use ensure that you are catching and throwing a known-to-pickify-team error